docs/Getting Started/Fedora/Fedora Root On ZFS.rst

```diff
@@ -0,0 +1,753 @@
+
+     zfs create -o canmount=off -o mountpoint=none rpool/ROOT
+
+   On Solaris systems, the root filesystem is cloned and the suffix is
+   incremented for major system changes through ``pkg image-update`` or
+   ``beadm``. Similar functionality has not yet been implemented into Fedora and will most likely never be added to Fedora in the forseeable future due to licensing issues.
+
+#. Create filesystem datasets for the root and boot filesystems::
+
+     zfs create -o canmount=noauto -o mountpoint=/ rpool/ROOT/fedora
+     zfs mount rpool/ROOT/fedora
+
+   With ZFS, it is not normally necessary to use a mount command (either
+   ``mount`` or ``zfs mount``). This situation is an exception because of
+   ``canmount=noauto``.
+
+#. Create datasets::
+
+     zfs create                                 rpool/home
+     zfs create -o mountpoint=/root             rpool/home/root
+     zfs create -o canmount=off                 rpool/var
+     zfs create -o canmount=off                 rpool/var/lib
+     zfs create                                 rpool/var/log
+     zfs create                                 rpool/var/spool
+
+   The datasets below are optional, depending on your preferences and/or
+   software choices.
+
+   If you wish to exclude these from snapshots::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/cache
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/tmp
+     chmod 1777 /mnt/var/tmp
+
+   If this system will use Docker (which manages its own datasets &
+   snapshots)::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/lib/docker
+
+   If this system will use NFS (locking)::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/lib/nfs
+
+   A tmpfs is recommended later, but if you want a separate dataset for
+   ``/tmp``::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/tmp
+     chmod 1777 /mnt/tmp
+   
+   Note that the reason why we are not fully seperating everything like we did in Ubuntu is because dnf will fail to install or update certain packages if we create too many datasets.
+
+   The primary goal of this dataset layout is to separate the OS from user
+   data. This allows the root filesystem to be rolled back without rolling
+   back user data.
+
+   If you do nothing extra, ``/tmp`` will be stored as part of the root
+   filesystem. Alternatively, you can create a separate dataset for ``/tmp``,
+   as shown above. This keeps the ``/tmp`` data out of snapshots of your root
+   filesystem. It also allows you to set a quota on ``rpool/tmp``, if you want
+   to limit the maximum space used. Otherwise, you can use a tmpfs (RAM
+   filesystem) later.
+
+#. Copy the LiveCD to your HDD/SDD::
+
+     rsync -avxHASX / /mnt/
+   
+   It is important to not forget the trailing /.
+   This command copies the LiveCD to our new zfs datasets and this is the only way I have found to reliably install and boot Fedora Workstation
```

Comment:
what issues did you have during the installation of the `Fedora Workstation` packages group?

Reply:
System fails to boot and hangs at Starting GNOME Desktop Manager

Reply:
TTY also fails at this point and doesn't allow you to login